src/elm/Page/Community/ObjectiveEditor.elm

```diff
@@ -209,13 +213,21 @@ viewForm { shared } objForm =
                     ]
                     []
                 ]
-            , button
-                [ class "button button-primary"
-                , type_ "submit"
-                , onClick ClickedSaveObjective
-                , disabled isDisabled
+            , div [ class "flex justify-between space-x-4" ]
+                [ button
+                    [ class "button button-primary"
+                    , type_ "submit"
+                    , onClick ClickedSaveObjective
+                    , disabled isDisabled
+                    ]
+                    [ text (t "community.objectives.editor.submit") ]
+                , button
```

Comment:
this button should only be shown in the case of an edit, the way it is now it will show even during the creation of an objective